runtime/interpreter/primitivestatictype.go

```diff
@@ -189,8 +189,6 @@ const (
 	_
 	_
 	_
```

Comment:
These lines were accidentally removed and need to brought back, otherwise the values for the following constants are changed, which breaks existing data:

```suggestion
	_
	_
	_
```